### ISSUE TITLE
base64: true not work in Android, returns file:/// uri scheme

### ISSUE THREAD

**Issue by brcontainer** [protonet/cordova-plugin-image-resizer#67]
base64: true not work in Android, returns file:/// uri scheme, code:

```
            (<any>window).ImageResizer.resize({
                uri: url,
                quality: 90,
                width: 320,
                height: 240,
                base64: true
            }, (data) => {
                  console.log('RESPONSE', JSON.stringify(data));
            }, (err) => {
                  console.log('ERROR', JSON.stringify(err));
            });
```

Is returned a url like this: `file:///storage/emulated/0/1554074577205.jpg`

**Comment by jvkxhola** [comment]
Bad news. Apparently, returning base64 is only possible on IOS

If you check the header it says:
```
    /**
     * Whether or not to return a base64 encoded image string instead of the path to the resized image.
     * iOS only
     */
    base64?: boolean;
```

**Comment by brcontainer** [comment]
@jvkxhola you're right, the problem is that you already have support for Android, see: https://github.com/JoschkaSchulz/cordova-plugin-image-resizer/commit/100e8748610dab4e0b6afde7d1315a7396339585 (31 Jul 2018)

But so far the author has not published the version "0.2.1" or "0.3.0", it is only in the **master branch.**

_See code details:_

- https://github.com/JoschkaSchulz/cordova-plugin-image-resizer/blob/master/src/android/ImageResizer.java#L70
- https://github.com/JoschkaSchulz/cordova-plugin-image-resizer/blob/master/src/android/ImageResizer.java#L119

> Related: https://github.com/JoschkaSchulz/cordova-plugin-image-resizer/issues/66

**Comment by mvzulu** [comment]
Hi guys, ran into the same problem, getting an update to this any time soon? thanks

**Comment by brcontainer** [comment]
> Hi guys, ran into the same problem, getting an update to this any time soon? thanks

temporarily try: https://github.com/JoschkaSchulz/cordova-plugin-image-resizer/issues/66
